lib/traject/config/folio_config.rb

```diff
@@ -32,6 +32,7 @@
     require 'traject/readers/folio_postgres_reader'
     provide 'reader_class_name', 'Traject::FolioPostgresReader'
   else
+    require 'traject/readers/folio_json_reader'
```

Comment:
it might be dumb to eagerly require this, but I wasn't sure how to detect that traject was running with `--stdin`

Reply:
fixed by [2cf1178](https://github.com/sul-dlss/searchworks_traject_indexer/pull/816/commits/2cf1178f4c1ee02005d19d6ba29b868ccddceaf8)